searx/webapp.py

```diff
@@ -265,6 +265,9 @@ def custom_url_for(endpoint: str, **values):
 
 
 def morty_proxify(url: str):
+    if not url:
```

Comment:
Are we still compatible with Morty somehow or is this just yet an other leftover?

Reply:
I have never tested if it works, but it seems there are still users of this feature:

- https://github.com/searxng/searxng/issues/3888#issuecomment-2684588571